scripts/utils/internals.js

```diff
@@ -174,15 +168,33 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     return web3.utils.padLeft(await web3.eth.getStorageAt(safeAddress, fallbackHandlerStorageSlot), 40)
   }
 
+  const getSafeCompatibleSignature = async function (transactionHash, signer) {
+    const sig = await web3.eth.sign(transactionHash, signer)
+    console.log(`Modifying recovery byte (${sig.slice(-2)}) for compatibility with Gnosis Safe Transaction Services`)
```

Comment:
Is this log statement still needed? It feels like this should be a comment rather than a print statement

Reply:
Right. No longer necessary. I had this here so we might be able to detect if there were other values we didn’t account for. This was essentially my hack for not understanding what could happen.